### ISSUE TITLE
Not been able to create regional bucket

### ISSUE THREAD

**Issue by Ss081994** [terraform-google-modules/terraform-google-cloud-storage#155]
### TL;DR

We have been using google cloud storage module to create a sample bucket . We are trying to create a regional bucket in us-east1 but the terraform code is throwing an error. Attached FYR

 Error: googleapi: Error 400: Project 982353099527 may not create storageClass MULTI_REGIONAL buckets with locationConstraint US-EAST1., invalid
│
│ with module.cloud_storage.google_storage_bucket.buckets["pcrf-01"],
│ on .terraform/modules/cloud_storage/main.tf line 40, in resource "google_storage_bucket" "buckets":
│ 40: resource "google_storage_bucket" "bucket" {
│

### Expected behavior

We have created a regional bucket before using this module. The version was 2.2.0 . The expected behaviour is it should allow the user to choose whether to create a regional or multi regional bucket. 

### Terraform Configuration

```hcl
module "cloud_storage" {
  source     = "terraform-google-modules/cloud-storage/google"
  version    = ">= 2.2.0"
  project_id = "imperial-terra-343108"
  prefix     = "bkt-nt-dev"
  location   = "us-east1"  
  storage_class = "regional"

  labels = {
    environment = "dev"
  }
```

### Terraform Version

```sh
1.1.7
```

**Comment by morgante** [comment]
What does your Terraform plan show?

**Comment by Xemnas0** [comment]
Same problem, my output is the following
`googleapi: Error 400: Project XXXXXXXXXXX may not create storageClass MULTI_REGIONAL buckets with locationConstraint EUROPE-WEST1., invalid`
